oauth-bundle/src/ContaoOAuthBundle.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\OAuthBundle;
```

Comment:
```suggestion
namespace Contao\OauthBundle;
```

In camel-case names it has to be written as `Oauth` IMO, otherwise snake-casing it would result in `o_auth_bundle` instead of `oauth_bundle`.

Reply:
That's already taken care of. The name of the bundle is `ContaoOAuthBundle`, but its alias will be `contao_oauth`, not `contao_o_auth` (which it would be by default). See 

https://github.com/contao/contao/blob/d21c3f9805eed522d152d169a351baf49d23765c/oauth-bundle/src/DependencyInjection/ContaoOAuthExtension.php#L33-L36

`knpuniversity/oauth2-client-bundle` handles it the same way. Its bundle's name is `KnpUOAuth2ClientBundle` but its alias is `knpu_oauth2_client` and not `knp_u_o_auth2_client`.

Personally I prefer to keep `OAuth` in the FQCN instead of `Oauth`. But that's of course highly subjective.

Reply:
Personally I (strongly 😅) prefer to have strict naming rules for camel case names, and that is IMO that the casing of the base words is strictly ignored. We also lowercase abbreviations like XML or HTTP.

But I was not able to find such a rule in the symfony coding style, so it is subjective, and I’m fine either way ☺️